src/dvclive/studio.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+
+from dvclive.utils import parse_scalars
+
+
+def _filter_plot_updates(plot, latest_step):
+    return [x for x in plot if int(x["step"]) >= latest_step]
+
+
+def _to_dvc_format(data):
+    formatted = {}
+    for k, v in data.items():
+        formatted[k] = {"data": v}
+    return formatted
+
+
+def _get_updates(live):
+    plots, metrics = parse_scalars(live)
+    latest_step = live._latest_studio_step
+
+    for name, plot in plots.items():
+        plots[name] = _filter_plot_updates(plot, latest_step)
+
+    metrics = {live.summary_path: {"data": metrics}}
+    plots = _to_dvc_format(plots)
+    return metrics, plots
+
+
+def post_to_studio(live, type):
+    import requests
+
+    timestamp = datetime.now().isoformat()
+    if type == "data":
+        metrics, plots = _get_updates(live)
+    else:
+        metrics = plots = {}
+
+    return requests.post(
+        "https://0.0.0.0",
```

Comment:
@Suor What would be the endpoint?

Reply:
Most probably `https://studio.iterative.ai/api/live` for production. Staging `https://dev.iterative.ai/api/live`, local `http://localhost:8111/api/live`.